source/jormungandr/jormungandr/realtime_schedule/sytral.py

```diff
@@ -129,6 +129,10 @@ def _make_url(self, route_point):
         for stop_id in stop_id_list[1:]:
             url += "&stop_id={stop_id}".format(stop_id=stop_id)
 
+        direction_type = route_point.fetch_direction_type()
+        if direction_type:
+            url += "&direction_type={direction_type}".format(direction_type=direction_type)
```

Comment:
Just to confirm: this is not our business to check how data is filled for `direction_type` in NTFS ?
We will just get a 400 from subservice and that's it if `direction_type=wtf`.

Reply:
Jormun acts like a hatch. It just transmit the value contained into data. But currently, this is in the GO PR that is protected.